packages/core/src/shipping/strategies/paypal-commerce/paypal-commerce-accelerated-checkout-shipping-strategy.ts

```diff
@@ -0,0 +1,175 @@
+import {
+    isPayPalCommerceAcceleratedCheckoutCustomer,
+    PayPalCommerceAcceleratedCheckoutUtils,
+    PayPalCommerceConnectAuthenticationState,
+    PayPalCommerceConnectStylesOption,
+} from '@bigcommerce/checkout-sdk/paypal-commerce-utils';
+
+import { AddressRequestBody } from '../../../address';
+import { BillingAddressActionCreator } from '../../../billing';
+import { CheckoutStore, InternalCheckoutSelectors } from '../../../checkout';
+import { InvalidArgumentError } from '../../../common/error/errors';
+import { PaymentMethodActionCreator } from '../../../payment';
+import { PaymentProviderCustomerActionCreator } from '../../../payment-provider-customer';
+import ConsignmentActionCreator from '../../consignment-action-creator';
+import { ShippingRequestOptions } from '../../shipping-request-options';
+import ShippingStrategy from '../shipping-strategy';
+
+import PayPalCommerceAcceleratedCheckoutShippingInitializeOptions from './paypal-commerce-accelerated-checkout-shipping-initialization-options';
+
+export default class PayPalCommerceAcceleratedCheckoutShippingStrategy implements ShippingStrategy {
+    constructor(
+        private _store: CheckoutStore,
+        private _billingAddressActionCreator: BillingAddressActionCreator,
+        private _consignmentActionCreator: ConsignmentActionCreator,
+        private _paymentMethodActionCreator: PaymentMethodActionCreator,
+        private _paymentProviderCustomerActionCreator: PaymentProviderCustomerActionCreator,
+        private _paypalCommerceAcceleratedCheckoutUtils: PayPalCommerceAcceleratedCheckoutUtils,
+    ) {}
+
+    updateAddress(
+        address: AddressRequestBody,
+        options?: ShippingRequestOptions,
+    ): Promise<InternalCheckoutSelectors> {
+        return this._store.dispatch(this._consignmentActionCreator.updateAddress(address, options));
+    }
+
+    selectOption(
+        optionId: string,
+        options?: ShippingRequestOptions,
+    ): Promise<InternalCheckoutSelectors> {
+        return this._store.dispatch(
+            this._consignmentActionCreator.selectShippingOption(optionId, options),
+        );
+    }
+
+    deinitialize(): Promise<InternalCheckoutSelectors> {
+        return Promise.resolve(this._store.getState());
+    }
+
+    async initialize(
+        options: PayPalCommerceAcceleratedCheckoutShippingInitializeOptions,
+    ): Promise<InternalCheckoutSelectors> {
+        const { methodId, styles } = options || {};
+
+        if (!methodId) {
+            throw new InvalidArgumentError(
+                'Unable to proceed because "methodId" argument is not provided.',
+            );
+        }
+
+        // Should run Authentication flow
```

Comment:
Do we need all these comments?

Reply:
For reference https://bpoplauschi.github.io/2021/01/20/Clean-Code-Comments-by-Uncle-Bob-part-2.html

Reply:
I will clean all comments in the file